Functions.Templates/Templates/MySqlOutputBinding-CSharp-Isolated/.template.config/template.json

```diff
@@ -47,7 +47,8 @@
           "ManualInstructions": [],
           "args": {
             "referenceType": "package",
-            "reference": "Microsoft.Azure.Functions.Worker.Extensions.MySql", "version": "0.0.000",
+            "reference": "Microsoft.Azure.Functions.Worker.Extensions.MySql",
+            "version": "1.0.2-preview",
```

Comment:
major version change?

Reply:
I think `0.0.000` is just a placeholder that was never updated with the version of "MySql" package when it became available. I validated the template with this change and didn't see any issues. (same for comment above for the csharp proj)

Reply:
Also noticed that with this set to v0, the extension is never loaded so the project doesn't even work. Adding a real version here also fixes this issue.